package.json

```diff
@@ -39,7 +39,6 @@
     "jquery": "^3.3.1",
     "jsdom": "^13.1.0",
     "jsqr": "^1.1.1",
-    "lodash": "^4.17.5",
```

Comment:
@HarshKhandeparkar Any reason for removing lodash?

Reply:
No I will add it again. I did not realize that I removed it. 😊